docs/usage.md

```diff
@@ -74,6 +78,43 @@ RUN mkdir -p /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 COPY 50-weekly.conf /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 ```
 
+## Air-gapped and dissconnected updates
+
+For environments without a direct connection to a centralized container
+registry, we encourage mirroring a registry in DMZ if possible or manually
+moving container images using `skopeo copy`.
+
+Systems that are completely dissconnected and require manual updates via USB
+drives this is relatively simple.
+
+Copy image to USB Drive:
+
+```skopeo copy docker://[registry]/[path to image] dir://run/media/$USER/$DRIVE/$DIR```
+
+*note, Using the dir transport will create a number of files,
+and it's recommended to place the image in it's own directory.
+If the image is local the containers-storage transport will transfer
+the image from a system directly to the drive:
+
+```skopeo copy containers-storage:[image]:[tag] dir://run/media/$USER/$DRIVE/$DIR```
+
+From the client system, insert the USB drive and mount it:
+
+```mount /dev/$DRIVE /mnt```
+
+`bootc switch` will direct the system to look at this mount point for future
+updates, and is only necessary to run one time if you wish to continue
+consuming updates from USB devices. note that if the mount point changes,
+simply run this command to point to the alternate location. We recommend
+using the same location each time to simplfy this.
+
+```bootc switch --transport dir --retain /mnt/$DIR```
```

Comment:
Hmm, why `--retain`?  I'm thinking we should actually drop that option because the direction lately is to make the source of truth the bootloader entries.

Reply:
That's only from me misinterpreting the output of -h. Let's remove it!